src/System.ServiceModel.Tests.Common/src/BaseAddress.cs

```diff
@@ -5,6 +5,29 @@
 
 public static class BaseAddress
 {
+#if USE_FIDDLER
+    // Base address for testing nonexistent endpoints
```

Comment:
Fiddler is very helpful when debugging issues related to requests. I'd vote we check this in now and modify it when we have the long term solution.